deployer/src/deployer/upload.py

```diff
@@ -5,21 +5,20 @@
 from dataclasses import dataclass
 from functools import cached_property
 from itertools import chain
+from pathlib import Path
 
 import boto3
-from boto3.s3.transfer import S3TransferConfig
-
 import click
+from boto3.s3.transfer import S3TransferConfig
 
 from .constants import (
     DEFAULT_CACHE_CONTROL,
     HASHED_CACHE_CONTROL,
-    MAX_WORKERS_PARALLEL_UPLOADS,
     LOG_EACH_SUCCESSFUL_UPLOAD,
+    MAX_WORKERS_PARALLEL_UPLOADS,
```

Comment:
Apart from the new import, by me, these other two changes happened because I ran the vscode tool to sort imports. Sorry. 
I'm not a big fan of being a stickler with alphabetically sorted imports but it definitely has its advantages in that you get less chance of git merge conflicts if people don't just add their new imports to the bottom of a list.